readme.md

```diff
@@ -1,4 +1,4 @@
-# vulkan
+<img src="./haskell-vulkan.png" width="348" alt="Haskell Vulkan Logo">
```

Comment:
I think that would be `![](haskell-vulkan.png)` in markdown.

Reply:
that's better thanks.